src/LINEBot/HTTPClient/CurlHTTPClient.php

```diff
@@ -111,7 +125,7 @@ private function getOptions($method, $headers, $reqBody)
             CURLOPT_BINARYTRANSFER => true,
             CURLOPT_HEADER => true,
         ];
-        if ($method === 'POST') {
+        if ($method === 'POST' || $method === 'PUT') {
```

Comment:
It seems that CURLOPT_PUT is not set

Reply:
unnecessary to use `CURLOPT_PUT ` because `CURLOPT_CUSTOMREQUEST => $method` used.